site/src/pages/SetupPage/SetupPageView.tsx

```diff
@@ -70,14 +66,6 @@ export const SetupPageView: React.FC<SetupPageViewProps> = ({
       <Welcome message={Language.welcomeMessage} />
       <form onSubmit={form.handleSubmit}>
         <Stack>
-          <TextField
-            {...getFieldHelpers("organization")}
-            onChange={onChangeTrimmed(form)}
-            autoFocus
-            fullWidth
```

Comment:
It might be nice to give `autoFocus` to the next field so that users can start typing without clicking into the form.